### DISCUSSION TITLE
Welcome to django-heralder Discussions!

### DISCUSSION THREAD

**Discussion by peterfarrell** [peterfarrell/django-heralder#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vanderzielj** [comment]
Hi there,

I learned about the django-heralder project on the issue board for [django-auth-adfs](https://github.com/snok/django-auth-adfs/pull/385) where you just provided the update to enable support for Django 5.2 (thank you!). I would like to know how the functionality that Django Heralder provides fits into the Django feature set - particularly does it extend or replace or add on to existing functionality? Ultimately I am wondering if it replaces or is adjacent to the functionality provided by the [django-o365](https://pypi.org/project/django-o365mail/) python library. I am using django-auth-adfs and I only recently added the [django-o365](https://pypi.org/project/django-o365mail/) library to my Django app. The django-o365 libary depends heavily on the python-o365 library and extends it to enable (easier) integration into a Django project (I do not contribute nor maintain this library; this how I would describe the library) to enable the sending of mail via Microsoft's Graph API. The django-o365 python library wraps the Django core mail send functionality with a call to the Oauth2 Graph API. 

About me: I am a full-stack developer working on a solution for my employer to enable our business partners to manage the configuration of their IoT devices purchased from us and view their data that we analyze for them on our platform. This is not a commercial product at this point though I am developing it for a publicly traded company. Though I have more than 15 years dev experience, have been developing using python and Django for less than two years now.

Regards,

Justin
